pandas/core/indexes/accessors.py

```diff
@@ -328,19 +334,19 @@ def to_pydatetime(self) -> np.ndarray:
         1   2018-03-11
         dtype: datetime64[ns]
 
-        >>> s.dt.to_pydatetime()
+        >>> s.dt.to_pydatetime()  # doctest: +SKIP
```

Comment:
I think I would prefer to keep these examples running, so that when we change the behavior we are reminded to change these examples

Reply:
Ok, I can see that point. Keep the deprecation notice?

Reply:
> Keep the deprecation notice?

Yes definitely